packages/babel-plugin-transform-wpcalypso-async/test/index.js

```diff
@@ -33,8 +33,8 @@ describe( 'babel-plugin-transform-wpcalypso-async', () => {
 				const code = transform( 'export default () => <AsyncLoad require="foo" />;' );
 
 				expect( code ).toBe(
-					'var _ref = () => import( /*webpackChunkName: "async-load-foo"*/"foo");\n' +
-						'export default (() => <AsyncLoad require={_ref} />);'
+					'var _ref = () => import(/*webpackChunkName: "async-load-foo"*/"foo");\n' +
+						'export default () => <AsyncLoad require={_ref} />;'
```

Comment:
Some formatting differences in the output. Nothing is functionally different so I updated the tests results to match the new output.